### DISCUSSION TITLE
Migration from one server to another

### DISCUSSION THREAD

**Discussion by cerri** [minio/minio#14323] [Q&A] [answered]
During our tests, we've delevoped a minio server with just one disk (on raid software).
Now we wish to migrate the full installation (data and configuration) to a new server, but with a different disk configuration.

In discussion we saw to use mc mirror: is it the best scenario to accomplish this move? How can we move the server configuration?

Thank you

**Comment by harshavardhana** [comment, marked as answer]
> In discussion we saw to use mc mirror: is it the best scenario to accomplish this move? How can we move the server configuration?

for data `mc mirror` and for the rest use other relevant commands `mc admin ...`
